### PR TITLE
adding translation of datetime and location names

### PR DESCRIPTION
**Pull request by tcarlsen**
I'm not english, therefore `prefer_english_names` dont help me, but I wanted to get location and datetime *(month name)* in danish.

I said this in my other PR aswell: Im no python developer so please let me know if this is the wrong way to do things

### CODE REVIEW

File: elodie/filesystem.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import re
 import shutil
 import time
+import locale
```
Comment (jmathai) [inline elodie/filesystem.py]: Can these changes go into geolocation.py instead of filesystem.py?

Comment (tcarlsen) [reply elodie/filesystem.py]: but its in filesystem.py `strftime` is used?

**Comment by tcarlsen** [comment]
Im sorry. I dont get this testing in python apparently.. 😢

**Comment by tcarlsen** [comment]
I dont get why googlephotos tests now are failing. I think im throwing in the towel before I mess more things up

File: elodie/filesystem.py (hunk 1 above)
Comment (tcarlsen) [reply elodie/filesystem.py]: locale is imported to translate `strftime` *(mostly month names)* and that is happening in filesystem.py

**Comment by tcarlsen** [comment]
okay why is this faling??

```
test_lookup_with_prefer_language_danish
1983    assert res['address']['city'] == 'K\xf8benhavn', res
1984AssertionError: {u'display_name': u'18G, Vestergade, K\xf8dbyen, Vesterbro, K\xf8benhavn, K\xf8benhavns Kommune, Region Hovedstaden, 1456, Danmark', u'place_id': u'1675725', u'lon': u'12.568567', u'boundingbox': [u'55.677547', u'55.677747', u'12.568467', u'12.568667'], u'osm_type': u'node', u'licence': u'Data \xa9 OpenStreetMap contributors, ODbL 1.0. https://osm.org/copyright', u'osm_id': u'340565033', u'lat': u'55.677647', u'address': {u'city': u'K\xf8benhavn', u'house_number': u'18G', u'country': u'Danmark', u'county': u'K\xf8benhavns Kommune', u'suburb': u'Vesterbro', u'state': u'Region Hovedstaden', u'road': u'Vestergade', u'country_code': u'dk', u'neighbourhood': u'K\xf8dbyen', u'postcode': u'1456'}}
```

I test for address city to be "K\xf8benhavn" and the result is "K\xf8benhavn".... why is that a failed test?

**Comment by tcarlsen** [comment]
only python 2.7 is failing now.. not sure how to fix... sorry all the spam commits

**Comment by tcarlsen** [comment]
been a year. any1 wanna help me fix the last error so this translation pr can be part of master? @jmathai

**Pull request closed without merging**